subsys/settings/src/settings_fcb.c

```diff
@@ -38,6 +38,11 @@ static const struct settings_store_itf settings_fcb_itf = {
 	.csi_save = settings_fcb_save,
 };
 
+__weak int settings_fcb_get_flash_area(void)
```

Comment:
Add description, to this function, why it is provided and why this is weak function.
Without this information it may fall victim of refactoring, at some point.

Reply:
Here would be enough.